cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/model/CVRSvVariant.java

```diff
@@ -159,36 +165,43 @@ public CVRSvVariant() {
     }
 
     /**
-    *
-    * @param pairedEndReadSupport
-    * @param site1Pos
-    * @param svLength
-    * @param eventInfo
-    * @param site2Desc
-    * @param svVariantId
-    * @param splitReadSupport
-    * @param normalReadCount
-    * @param svClassName
-    * @param breakpointType
-    * @param site2Pos
-    * @param normalVariantCount
-    * @param tumorVariantCount
-    * @param tumorReadCount
-    * @param confidenceClass
-    * @param site1Chrom
-    * @param connectionType
-    * @param site1Gene
-    * @param svDesc
-    * @param connType
-    * @param mapq
-    * @param variantStatusName
-    * @param site1Desc
-    * @param comments
-    * @param site2Chrom
-    * @param site2Gene
-    * @param annotation
-    */
-    public CVRSvVariant(String annotation, String breakpointType, String comments, String confidenceClass, String connType, String connectionType, String eventInfo, String mapq, String normalReadCount, String normalVariantCount, String pairedEndReadSupport, String site1Chrom, String site1Desc, String site1Gene, String site1Pos, String site2Chrom, String site2Desc, String site2Gene, String site2Pos, String splitReadSupport, String svClassName, String svDesc, String svLength, String svVariantId, String tumorReadCount, String tumorVariantCount, String variantStatusName) {
+     * @param annotation
```

Comment:
this was auto-generated from IntelliJ, I don't think it hurts to update it? don't think it's being used anywhere